automerge/src/lib.rs

```diff
@@ -327,11 +340,18 @@ impl Automerge {
         prop: P,
         value: i64,
     ) -> Result<OpId, AutomergeError> {
-        self.local_op(obj.into(), prop.into(), OpType::Inc(value))
+        match self.local_op(obj.into(), prop.into(), OpType::Inc(value))? {
+            Some(opid) => Ok(opid),
+            None => { panic!("increment should always create a new op") }
+        }
     }
 
     pub fn del<P: Into<Prop>>(&mut self, obj: OpId, prop: P) -> Result<OpId, AutomergeError> {
-        self.local_op(obj.into(), prop.into(), OpType::Del)
+        // TODO: Should we also no-op multiple delete operations?
+        match self.local_op(obj.into(), prop.into(), OpType::Del)? {
+            Some(opid) => Ok(opid),
+            None => { panic!("increment should always create a new op") }
```

Comment:
I think this error message is wrong.

Reply:
You think we could sometime have a noop increment?

Reply:
Oh, I see what you mean :flushed:

Reply:
Fixed.